rfc/rfc004-verifiable-transactional-graph.md

```diff
@@ -54,21 +55,28 @@ In addition to required header parameters as specified in RFC7515 the following
 
 * **cty**: MUST contain the type of the transaction content indicating how to interpret it.
 * **crit** MUST contain the **sigt**, **ver**, **prevs** and **pal** headers.
```

Comment:
Question, should pal always be present, or only when there is a pal header?

Reply:
yes